controlplane/package.json

```diff
@@ -17,6 +17,7 @@
     "deactivate-org": "tsx src/bin/deactivate-org.ts",
     "reactivate-org": "tsx src/bin/reactivate-org.ts",
     "seed:billing": "tsx src/bin/seed-billing.ts",
+    "migrate-groups": "tsx src/bin/migrate-groups.ts",
```

Comment:
This makes it look like that it something we need on a regular basis. I'd remove it and having the reference in `data_migration` is enough.